lib/linguist/languages.yml

```diff
@@ -891,7 +891,7 @@ Creole:
   language_id: 71
 Crystal:
   type: programming
-  color: "#776791"
+  color: "#000"
```

Comment:
Could you try adding the last three implicit zeros? I think the Travis CI tests are failing because of that.

Reply:
Yeah, I'd say it is too:

~~~
  1) Error:
TestColorProximity#test_color_proximity:
TypeError: no implicit conversion of nil into String
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:35:in `+'
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:35:in `split_color'
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:30:in `proximity_of'
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:15:in `block in past_threshold?'
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:14:in `each'
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:14:in `each_with_object'
    /home/travis/build/github/linguist/vendor/bundle/ruby/2.4.0/gems/color-proximity-0.2.1/lib/color-proximity.rb:14:in `past_threshold?'
    /home/travis/build/github/linguist/test/test_color_proximity.rb:14:in `block in test_color_proximity'
    /home/travis/build/github/linguist/test/test_color_proximity.rb:13:in `map'
    /home/travis/build/github/linguist/test/test_color_proximity.rb:13:in `test_color_proximity'
~~~